site/docs/resources/faq.md

```diff
@@ -26,6 +27,42 @@ No, it's not.
    As always, TypeScript will auto-complete the parameter names for you.
 3. Perhaps double-check the method signature for [actions](../guide/context#available-actions) on `ctx` [here](https://deno.land/x/grammy/mod.ts?s=Context#Methods), or for API methods (`ctx.api`, `bot.api`) [here](https://deno.land/x/grammy/mod.ts?s=Api#Methods).
 
+## How Can I Access the Chat History?
+
+You can't.
+
+Telegram does not store the messages for your bot.
```

Comment:
This as it is phrased right now, is right on the high level, but also a lie if you take it seriously. Would you like to go with it?

Reply:
I think that the people who read this basic question do actually need to hear a simplified view of the things. Otherwise they start hacking around with `forwardMessage` to private channels etc, and those are big hacks that don't scale well and potentially get people banned.

How about this?
```suggestion

Telegram does not store the messages for your bot (although messages are of course stored for the user).
```

Reply:
```suggestion

Telegram does not let bots see chat histories.
```

How about this one?

Reply:
That is more confusing than not saying this at all. It sounds like they just have some obscure reason why they don't reveal the data. But that is not the case, is it?